ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/precompiles/AbstractPrecompiledContractTest.java

```diff
@@ -36,7 +36,7 @@ public AbstractPrecompiledContractTest(
         registryFactory
             .apply(
                 new PrecompiledContractConfiguration(
-                    new BerlinGasCalculator(), PrivacyParameters.DEFAULT))
```

Comment:
I'm not sure about this, would love other opinions, but do we want to still have a `BerlinGasCalculator` that just delegates to `IstanbulGasCalculator`? I think it makes it more obvious which parts of the code have been updated if we use this convention.

Reply:
That thought crossed my mind.  However I think making a no-op gas calculator might imply to the casual reader that there were gas cost changes in Berlin when in fact there are not currently.

If the two gas cost EIPs do make an appearance we can return it.

Reply:
I also had a doubt regarding this part. but I think that if we don't change anything it's not useful to have a GasCalculator per fork but only when we change something IMO